### PR TITLE
Fix Issue #889 - Dashboard Style Refinement

### PR DESCRIPTION
**Pull request by startailcoon**
## Fixes Issue #889 
- Smother Dashboard Tabs colors
- Moving to full width for content
- Added padding on sections required so as max width has been removed.

## Partially merged with PR 1140
Taken parts of the Header.jsx, _header.scss, _theme-template.scss to incorperate dynamic header.
Complies with PR.

### CODE REVIEW

**Comment by wmbutler** [comment]
Latest pull from this PR shows console errors. I spoke with @startailcoon and we are hoping you can find something obvious @svk31 

```
warning.js:33 Warning: Each child in an array or iterator should have a unique "key" prop. Check the render method of `Header`. See https://fb.me/react-warning-keys for more information.
    in li (created by Header)
    in Header (created by StatefulHeaderContainer)
    in StatefulHeaderContainer (created by App)
    in div (created by App)
    in div (created by App)
    in div (created by App)
    in App (created by RootIntl)
    in IntlProvider (created by RootIntl)
    in RootIntl (created by StatefulRootIntlContainer)
    in StatefulRootIntlContainer (created by Root)
    in Root (created by Unknown)
    in Unknown (created by RouterContext)
    in RouterContext (created by Router)
    in Router (created by Routes)
    in Routes
```

**Comment by startailcoon** [comment]
Waiting to see @svk31 fixes on PR 1140 for this issue.

**Comment by zhyupe** [comment]
Are the structures and stylesheets determined? If so, I can modify my code per this.

**Comment by svk31** [comment]
I'd appreciate it if you could merge the latest staging into your branch and fix the css conflicts. There are quite a few, and I'm not sure all of them are intentional or if the merge is just messing things up.

File: app/components/Layout/Header.jsx [hunk 1]
```diff
@@ -252,7 +252,7 @@ class Header extends React.Component {
 
     render() {
         let {active} = this.state;
-        let {currentAccount, starredAccounts, passwordLogin, passwordAccount, height} = this.props;
```
Comment (zhyupe) [inline app/components/Layout/Header.jsx]: This will cause the following error
image

Comment (startailcoon) [reply app/components/Layout/Header.jsx]: Thanks. Must have happened when I merged with staging. Had another strange change that didn't come over with it.

**Comment by svk31** [comment]
Is it supposed to look like this?:

image

image

**Comment by startailcoon** [comment]
Bill wanted the same background as the tab. Maybe the top market could change since it has the man tab color.

I also see it needs some shifting in colour to match the new background.

**Comment by ahdigital** [comment]
How does one access the landing page with the login, create account, restore backup etc. buttons?

I used to clear all cache, local storage, cookies etc. and then it would appear, but after pulling updates from staging I am redirected to the dashboard.

**Pull request merged**